.github/workflows/image.yaml

```diff
@@ -111,6 +125,12 @@ jobs:
         dist:
           - ubuntu22.04
           - ubuntu24.04
+        base_target:
+          - jammy
+          - noble
```

Comment:
IMO defining both `base_target` and `dist` is redundant since there is a one-to-one mapping between the two of them. If we really need to define both strings, then one of them should be assigned in a Makefile or script.